src/platforms/platform.dom.css

```diff
@@ -19,6 +19,7 @@
 .chartjs-size-monitor-expand,
 .chartjs-size-monitor-shrink {
 	position: absolute;
+	direction: ltr;
```

Comment:
@kurkle do you know why adding this property makes it work?

Reply:
I did not extensively test, but I guess its because of (and setting scollLeft to 1000000):

>If the element's direction is rtl (right-to-left), then scrollLeft is 0 when the scrollbar is at its rightmost position (at the start of the scrolled content), and then increasingly negative as you scroll towards the end of the content.

[MDN:scrollLeft](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeft)